models/model.js

```diff
@@ -11,7 +11,14 @@ exports.selectArticleByID = (article_id) => {
       if (rows.length === 0) {
         return Promise.reject({ status: 404, msg: "article does not exist" });
       }
-      console.log(rows);
       return rows[0];
     });
 };
+
+exports.selectArticles = () => {
+  return db.query(`SELECT articles.*, COUNT(comments.comment_id) AS comment_count
```

Comment:
from here you should only send back the rows property and not the whole response object that comes back

Reply:
Remember this will also impact how you access the data in your tests